src/App Modules/Subscriptions/subscriptions.controller.ts

```diff
@@ -0,0 +1,54 @@
+/*eslint-disable*/
+import { Body, Controller, Get, ParseIntPipe, Post, Put, Query, Res } from "@nestjs/common";
+import { SubscriptionService } from "./Services/subscriptions.service";
+import { SubscriptionActivationService } from "./Services/subscriptionactivation.service";
+import { LoggedInUser } from "src/Shared Modules/Decorators/userid.decorator";
+import { Response } from "express";
+import { SubscriptionPlanDTO } from "./DTOs/SubscriptionPlanDTO";
+import { Role } from "src/App Modules/Subscriptions/Enums/roles.enum";
+import { Roles } from "src/Shared Modules/Decorators/role.decorator";
+
+@Controller('subscriptions')
+export class SubscriptionsController {
+    constructor(
+        private readonly subscriptionsService: SubscriptionService,
+        private readonly subscriptionActivationService: SubscriptionActivationService
+    ) { }
+
+    @Get()
+    async getSubscriptions(
+        @LoggedInUser() userid: number,
+        @Res() res: Response
+    ) {
+        const result = await this.subscriptionsService.getSubscriptionsOfUser(userid);
+        res.status(200).json(result);
```

Comment:
no need to set a status code, because the `GET` decorator by default returns 200
read this https://docs.nestjs.com/controllers#status-code